MCA/api/routes/staff/index.ts

```diff
@@ -30,4 +34,51 @@ staffRouter.get("/categories/:year", validatePhaseYear, async (ctx) => {
     ctx.body = categories.map(x => x.getInfo());
 });
 
+// Endpoint for granting direct MCA nom/vote access to users
+staffRouter.post("/grant/:year", validatePhaseYear, async (ctx) => {
+    if (!ctx.request.body.user)
+        return ctx.body = { error: "No user ID given!" };
+    if (!ctx.request.body.mode)
+        return ctx.body = { error: "No mode given!" };
+    if (/(standard|taiko|fruits|mania|storyboard)/.test(ctx.request.body.mode))
```

Comment:
add ^ and $ otherwise i could pretend `eruyfgyhbeugberugntaikoretugyeruy` is a valid mode

Reply:
yea